apps/etterlatte-samordning-vedtak/.nais/dev.yaml

```diff
@@ -137,6 +139,9 @@ spec:
           permissions:
             roles:
               - les-oms-vedtak
+        - application: tokenx-token-generator
+          namespace: aura
+          cluster: dev-gcp
```

Comment:
Denne var ny for meg. Fiffig! 
Hvis noen andre er nysgjerrige er det mer info her: https://github.com/nais/wonderwalled

Burde kanskje slenge på en kommentar som forteller at denne _ikke må kopieres til prod.yaml_